README.md

```diff
@@ -8,13 +8,73 @@
 + Delete a key-value pair from an object
 + Iterate over key-value pairs in an object
 
-
 ## What the Student Already Knows
 
 - Variables
 - Data Type- Strings and Integers
 - Functions
-- Booleans, Comparison Operators, Flow Control 
+- Booleans, Comparison Operators, Flow Control
 - Arrays
 
+##Understanding Objects
+
+JavaScript objects are a set of key-value pairs. Objects use the JavaScript object notation, for example, `{ 'key': 'value' }`. Each key inside an object may have any value type. For example, strings, integers, booleans, and functions. JavaScript does not care what the value is as long as it maps to a key in the set.
+
+To create an object, fire up a plain old REPL in Node.js. A REPL is a Read Eval Print Loop, and I invite you to type right along with me. Letʼs write this like so:
+
+```javascript
+> var o = { key1: 1 };
+```
+
+Notice in this example I omit the single quotes around the key when creating this object. JavaScript is not picky about syntax. For objects with special characters in the key, put single or double quotes around it. For example, `{ 'my-special-key': 'value' }`. I prefer to use single quotes, the choice is yours.
+
+###Accessing Objects
+
+To access the object I created above in the REPL, do this:
+
```

Comment:
is this accessing the object or accessing the value? why would we need to do that? this is where using narrative to explain why we're doing what we're doing would be helpful.